pyglmnet/pyglmnet.py

```diff
@@ -505,14 +525,25 @@ def pseudo_R2(self, y, yhat, ynull):
 
         elif self.distr == 'normal':
             R2 = 1 - np.sum((y - yhat)**2) / np.sum((y - ynull)**2)
+        elif self.distr == 'multinomial':
+            y = label_binarizer(y)
+            # yhat is the probability of each output
+            if yhat.ndim != y.ndim or ynull.ndim != y.ndim:
+                msg = 'yhat and ynull must be (n_samples, n_class) ndarrays'
+                raise Exception(msg)
```

Comment:
also, I just realized that you should not raise a generic `Exception` but a more specific one. See here for example: http://stackoverflow.com/a/24065533